### PR TITLE
Add Storez package

### PR DESCRIPTION
**Pull request by plrenaudin**

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit fad75ac05f0bc0f60fd3a724c993673c191e125c

https://deploy-preview-140--svelte-community.netlify.com

File: data/code.yml [hunk 1]
```diff
@@ -1739,3 +1739,11 @@ resources:
     tags:
       - components and libraries
       - inputs and widgets
+  - name: '`svelte-storez`'
```
Comment (ryanatkn) [inline data/code.yml]: Should be '\`storez\`' right?

File: data/code.yml [hunk 2]
```diff
@@ -1739,3 +1739,11 @@ resources:
...
+    url: 'https://github.com/plrenaudin/svelte-storez'
+    description: Add features to your Svelte writable store
+    tags:
+      - components and libraries
```
Comment (ryanatkn) [inline data/code.yml:1746]: also add the tag "stores and state"

File: data/code.yml [hunk 3]
```diff
@@ -1739,3 +1739,11 @@ resources:
     tags:
       - components and libraries
       - inputs and widgets
+  - name: '`svelte-storez`'
+    url: 'https://github.com/plrenaudin/svelte-storez'
+    description: Add features to your Svelte writable store
```
Comment (ryanatkn) [inline data/code.yml]: My personal preference is wording this differently but no worries if you disagree. I'd probably change it to "A writable store with extra features" or "A writable store with extra features including history tracking" (if that's the most important feature to call out)

**Review by ryanatkn — CHANGES_REQUESTED** [review]

**Review by ryanatkn — APPROVED** [review]

**Pull request merged**